packages/main/src/plugin/util/port.ts

```diff
@@ -55,12 +55,16 @@ export async function getFreePortRange(rangeSize: number): Promise<string> {
   return `${startPort}-${port - 1}`;
 }
 
-export function isFreePort(port: number): Promise<boolean> {
+function isFreeAddressPort(address: string, port: number): Promise<boolean> {
   const server = net.createServer();
   return new Promise((resolve, reject) =>
     server
       .on('error', (error: NodeJS.ErrnoException) => (error.code === 'EADDRINUSE' ? resolve(false) : reject(error)))
       .on('listening', () => server.close(() => resolve(true)))
-      .listen(port, '127.0.0.1'),
+      .listen(port, address),
   );
 }
+
+export async function isFreePort(port: number): Promise<boolean> {
+  return (await isFreeAddressPort('127.0.0.1', port)) && (await isFreeAddressPort('0.0.0.0', port));
```

Comment:
I'm wondering, should it try on all interfaces and not only on 127.0.0.1 and 0.0.0.0

let's imagine I'm connected to a router/wifi network, I'll have an ip and I can bind something on that port

and it'll tell me the port is free there no ?

Reply:
maybe something like

const networkInterfaces = [...];
const checkInterfaces = await Promise.all(networkInterfaces.map(interface => isFreeAddressPort(interface, port));
return checkInterfaces.every(element => element === true);

Reply:
This makes the tests fail on CI. It seems that listen fails on all ports of some interface in the CI's machine. Looking at it...